aytests/services.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+#set -e -x
+
+# Services have been enabled/disabled in the first installation stage
+# because on CaaSP there is no second stage anymore.
+
+OK=1
+
+zgrep "systemctl enable sshd.service" /var/log/YaST2/y2log-1.gz || OK=0
+zgrep "systemctl enable cloud-init-local.service" /var/log/YaST2/y2log-1.gz || OK=0
+zgrep "systemctl enable cloud-init.service" /var/log/YaST2/y2log-1.gz || OK=0
+zgrep "systemctl enable cloud-config.service" /var/log/YaST2/y2log-1.gz || OK=0
+zgrep "systemctl enable cloud-final.service" /var/log/YaST2/y2log-1.gz || OK=0
+zgrep "systemctl enable issue-generator.service" /var/log/YaST2/y2log-1.gz || OK=0
+zgrep "systemctl enable issue-add-ssh-keys.service" /var/log/YaST2/y2log-1.gz || OK=0
+zgrep "systemctl enable salt-minion.service" /var/log/YaST2/y2log-1.gz || OK=0
+zgrep "systemctl disable purge-kernels.service" /var/log/YaST2/y2log-1.gz || OK=0
```

Comment:
As long as you're using `set -e`, you don't need the `OK` variable. Just put one line below the other and the script will stop when some command fails (so `AUTOYAST OK` won't be printed).

```
zgrep "systemctl enable sshd.service" /var/log/YaST2/y2log-1.gz
# more zgreps

systemctl is-enabled sshd # it will return `1` if service is not enabled
```